pkg/registry/core/service/storage/rest_test.go

```diff
@@ -2280,3 +2312,79 @@ func TestUpdateNodePorts(t *testing.T) {
 		}
 	}
 }
+
+func TestServiceRegistryCreateWithNetworkOfExternalIPAM(t *testing.T) {
+	storage, registry, server := NewTestREST(t, nil)
+	storage.networks = &dummyNetworkGetter{ipamType: "External"}
+	defer server.Terminate(t)
+
+	svc := &api.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "foo",
+			Tenant:    "system",
```

Comment:
minor: use metav1.TenantSystem instead of "system".

Reply:
test data; simple string inplace seems to help understandability